contracts/src/L2/predeploys/L1GasPriceOracle.sol

```diff
@@ -90,12 +143,22 @@ contract L1GasPriceOracle is OwnableBase, IL1GasPriceOracle {
      *****************************/
 
     /// @inheritdoc IL1GasPriceOracle
-    function setL1BaseFee(uint256 _l1BaseFee) external override {
-        require(whitelist.isSenderAllowed(msg.sender), "Not whitelisted sender");
+    function setL1BaseFee(uint256 _l1BaseFee) external override onlyWhitelistedSender {
+        l1BaseFee = _l1BaseFee;
 
+        emit L1BaseFeeUpdated(_l1BaseFee);
+    }
+
+    /// @inheritdoc IL1GasPriceOracle
+    function setL1BaseFeeAndBlobBaseFee(
+        uint256 _l1BaseFee,
+        uint256 _l1BlobBaseFee
+    ) external override onlyWhitelistedSender {
         l1BaseFee = _l1BaseFee;
+        l1BlobBaseFee = _l1BlobBaseFee;
 
         emit L1BaseFeeUpdated(_l1BaseFee);
+        emit L1BlobBaseFeeUpdated(_l1BlobBaseFee);
```

Comment:
This event will be missing for the initial setting, not sure if that's a concern. Same for `CommitScalarUpdated` and `BlobScalarUpdated`.

Reply:
Not a big problem to me.